### PR TITLE
2nd Assignment by 202400363 (Seoyeon Hwang)

### PR DESCRIPTION
**Pull request by hwangseoyeon04**
# Assignment Submission

## Student Information
- **Student ID**: [202400363]
- **Name**: [Seoyeon Hwang]
- **Assignment**: [2nd Assignment]

## Changed Files

```
submissions
└── 202400363
    ├── assignment2
    │   ├── README.md
    │   ├── problem1
    │   │   ├── README.md
    │   │   └── main.py
    │   ├── problem2
    │   │   ├── README.md
    │   │   └── textops
    │   │       ├── __init__.py
    │   │       ├── clean
    │   │       │   ├── __init__.py
    │   │       │   └── filters.py
    │   │       ├── main.py
    │   │       └── tokenize
    │   │           ├── __init__.py
    │   │           └── word.py
    │   ├── problem3
    │   │   ├── README.md
    │   │   └── main.py
    │   ├── problem4
    │   │   ├── README.md
    │   │   └── dsops
    │   │       ├── __init__.py
    │   │       ├── main.py
    │   │       ├── split
    │   │       │   ├── __init__.py
    │   │       │   └── train_test.py
    │   │       └── stats
    │   │           ├── __init__.py
    │   │           └── labels.py
    │   ├── problem5
    │   │   ├── README.md
    │   │   └── cachekit
    │   │       ├── __init__.py
    │   │       └── main.py
    │   └── problem6
    │       ├── README.md
    │       └── main.py
    └── practice
```

### Problem 1
- [ ] Implemented required functionality
- [ ] All tests pass locally

### Problem 2
- [ ] Implemented required functionality
- [ ] All tests pass locally

### Problem 3
- [ ] Implemented required functionality
- [ ] All tests pass locally

### Problem 4 (Optional)
- [ ] Implemented required functionality
- [ ] All tests pass locally

### Problem 5 (Optional)
- [ ] Implemented required functionality
- [ ] All tests pass locally

### Problem 6 (Optional)
- [ ] Implemented required functionality
- [ ] All tests pass locally

## Testing
- [ ] I have run `python grader.py` locally
- [ ] All implemented problems pass their tests
- [ ] I have tested edge cases mentioned in the problem statements

## Additional Notes

---
**Note**: This PR will be automatically validated for:
- Correct directory structure
- Valid Python syntax
- PR title format: `Nth Assignment by {student_id} (Full Name)`
- File organization compliance

### CODE REVIEW

File: submissions/202400363/assignment2/problem2/textops/tokenize/word.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    Split on single spaces; empty/whitespace-only -> [].
+    Assume `s` is already normalized by clean_text.
+    """
+    # TODO: 구현하세요
+    # 힌트:
+    if not s or s.strip() == "":
+        return []
+
+    return s.split(" ")
+    # 3) 빈 리스트 반환 조건 잊지 말기
```
Comment (hist0613) [inline submissions/202400363/assignment2/problem2/textops/tokenize/word.py:12]: 쓸모없어진 코드는 잘 지워주세요~

File: submissions/202400363/assignment2/problem4/dsops/split/__init__.py [hunk 2]
```diff
@@ -0,0 +1 @@
+pass
```
[the remark this answers is not in the document]
Comment (hwangseoyeon04) [reply submissions/202400363/assignment2/problem4/dsops/split/__init__.py:1]: 이건 어떤 부분이 잘못된 걸까요?!

[the remark this answers is not in the document]
Comment (hist0613) [reply submissions/202400363/assignment2/problem4/dsops/split/__init__.py:1]: pass 가 없어도 됩니다 :)

**Review by hist0613 — COMMENTED** [review]
`grader.py` 기준 만점입니다. 
다만, 코드가 전체적으로 정돈이 안 되었다는 느낌이 있습니다.
가독성을 고려한 코드를 작성해주세요!

**Comment by hist0613** [comment]
```
Student ID: [Your Student ID]
Name: [Your Full Name]
Assignment: [Assignment Number]
```
본문 내용 채워주세요~
